blocks/explore-articles/explore-articles.js

```diff
@@ -1,20 +1,21 @@
-import { createElement, getTextLabel } from '../../scripts/common.js';
+import { createElement, getTextLabel, capitalizeWords } from '../../scripts/common.js';
 import {
-  fetchMagazineArticles,
-  getArticleTagsJSON,
-  removeArticlesWithNoImage,
+  fetchMagazineData,
+  formatArticlesArray,
+  formatFacetsArray,
 } from '../../scripts/services/magazine.service.js';
 
-const allArticles = await fetchMagazineArticles();
-const allArticlesWithImage = removeArticlesWithNoImage(allArticles);
-const allArticleTags = await getArticleTagsJSON();
-const allCategories = allArticleTags.categories;
-const allTrucks = allArticleTags.trucks;
 const blockName = 'explore-articles';
 
+const queryVariables = { facets: ['ARTICLE', 'TRUCK'], sort: 'LAST_MODIFIED_DESC' };
```

Comment:
there a few of magazines that have topics, so please add topics tags, no t just article-categories and trucks

Reply:
Can you define a more specific name for `allData`?